tests/cli.rs

```diff
@@ -53,6 +51,7 @@ mod cli {
         Ok(())
     }
 
+    #[cfg(target_family = "unix")]
```

Comment:
```suggestion
    #[cfg_attr(not(target_family = "unix"), ignore = "Ignored on non-unix")]
```

Same idea as the other comment, but with a slightly different incantation. That also requires that this test can compile on non-unix, but you can switch `create_soft_link()` back to its original impl, but swap out the body for windows to be `unimplmented!()` or something like that

Mostly I just want it to be obvious when tests are ignored so that people aren't confused with tests appearing/disappearing conditionally

Reply:
What's "that variant for failed jobs" exactly? Error::FailedJobs? They were collected from Replacer::replace_file runs, which were replaced with match arms.

Reply:
I agree on `#[ignore]`ing the test on just windows. Let's go with that :+1: